storm_control/steve/objectives.py

```diff
@@ -169,7 +169,7 @@ def addObjective(self, data):
                                  um_per_pixel = float(data[1]),
                                  x_offset = float(data[2]),
                                  y_offset = float(data[3]))
-        self.item_store.addItem(obj_item)
+        #self.item_store.addItem(obj_item)
```

Comment:
I commented this out because the item store had obj items as well as the points, which caused issues with how I implemented the feature for iterating through the points to draw the lines between them. Will we use the obj_items for anything?

Reply:
I don't necessarily foresee a need for it, if it's no longer needed you can delete it. Or Include in a comment why it was commented out

Reply:
Ok, thank you Collin!